streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -387,6 +391,12 @@
                     atLeast(0L),
                     ConfigDef.Importance.LOW,
                     RECONNECT_BACKOFF_MS_DOC)
+            .define(RECONNECT_BACKOFF_MAX_CONFIG,
+                    ConfigDef.Type.LONG,
+                    50L,
```

Comment:
@guozhangwang @enothereska @dguy Is it useful to apply an exponential backoff strategy for Streams?